src/Files.App/Views/LayoutModes/ColumnViewBase.xaml

```diff
@@ -176,14 +184,11 @@
 					</ListView.ItemContainerTransitions>
 
 					<ListView.ItemTemplate>
-						<DataTemplate x:DataType="local2:ListedItem">
+						<DataTemplate x:DataType="filesystem:ListedItem">
 							<Grid
 								x:Name="FilesRootGrid"
-								Height="30"
-								Margin="0"
-								Padding="12,0,12,0"
-								HorizontalAlignment="Stretch"
-								VerticalAlignment="Center"
+								Height="32"
```

Comment:
Height should be set to `30` in this case.

Reply:
Fixed